Debugging.md

```diff
@@ -0,0 +1,83 @@
+# Debugging your Game Servers
```

Comment:
>Debugging [](start = 2, length = 9)

We should clarify that they should not take any dependency on behavior or information they see while debugging the containers. Only publicly documented API and behavior are supported and other things can change without notice.

Reply:
Otherwise, folks will try to reverse engineer things that we don't want them to take a dependency on

---
In reply to: [446657105](https://github.com/PlayFab/gsdkSamples/pull/21#discussion_r446657105) [](ancestors = 446657105)